src/game/clock.h

```diff
@@ -7,3 +7,10 @@ bool Clock_Init();
 int32_t Clock_GetMS();
 int32_t Clock_Sync();
 int32_t Clock_SyncTicks(int32_t target);
+int32_t Clock_GetYear();
+int32_t Clock_GetMonth();
+int32_t Clock_GetDay();
+int32_t Clock_GetHours();
+int32_t Clock_GetMinutes();
+int32_t Clock_GetSeconds();
+char *Clock_GetDateTime(char *date_time);
```

Comment:
Missing newline character. 
Like mentioned earlier, most of `Clock_Get*` don't need to be public.